packages/app-mobile/components/NoteEditor/SearchPanel.tsx

```diff
@@ -33,6 +32,7 @@ export interface SearchPanelProps {
 
 interface ActionButtonProps {
 	styles: any;
+	theme: Theme;
```

Comment:
I'd rather we use themeId everywhere for consistency.

Reply:
We don't have access to the themeId here (without changes). Just the instantiated Theme object.

Reply:
Changed! Thank you for the feedback!